roles/magnum/templates/etc/magnum/magnum.conf

```diff
@@ -0,0 +1,224 @@
+[DEFAULT]
+
+debug = {{ magnum.logging.debug }}
+verbose = {{ magnum.logging.verbose }}
+log_dir = /var/log/magnum
+
+[api]
+
+# The port for the Magnum API server. (integer value)
+port = 9512
```

Comment:
none [or few] of the other projects currently have this.   There is a task being handled by ICOS to make all ports configurable ... so better to wait for that to happen and redo this as part of that effort ...  otherwise it will be implemented differently and need to be changed again.